community/dnscrypt-proxy/build

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -e
+
+cd dnscrypt-proxy
+
+export GOPATH="$PWD/go"
+export GO111MODULE=on
```

Comment:
Why?

Reply:
exporting `GOPATH` won't put `go`'s stuff in `$HOME/go`, saw both these variables used in other golang packages aswell so added them.

Reply:
These variables are no longer needed. `-mod=vendor` prevents cluttering $HOME

Reply:
The style guide could then be updated.
https://github.com/kiss-community/wiki/blob/master/kiss/style-guide.txt#L334

Reply:
Indeed it could. I'll make that adjustment soon(tm).

Reply:
https://github.com/kiss-community/wiki/pull/7